package.json

```diff
@@ -82,7 +82,8 @@
     "esnext": false,
     "rules": {
       "unicorn/prefer-optional-catch-binding": "off",
-      "unicorn/prefer-includes": "off"
+      "unicorn/prefer-includes": "off",
+      "node/no-deprecated-api": "warn"
```

Comment:
This one is controversial, however it will fail otherwise with 
{code}
  lib\util\resolve-base-url.js:8:16
  ×   8:16  url.parse was deprecated since v11.0.0. Use url.URL constructor instead.    node/no-deprecated-api

  lib\util\gh.js:15:12
  ×  15:12  url.resolve was deprecated since v11.0.0. Use url.URL constructor instead.  node/no-deprecated-api
{code}

Which is fair, but then we don't have any target set. I can try to detect, if modern API is present and use it. What you think?